generator/templates/generator/seed.html

```diff
@@ -1,9 +1,18 @@
 {% extends 'generator/base.html' %}
 
+{% block title %}Seed {{ share_id }} - {{ block.super }}{% endblock %}
+
 {% block imports %}
     {% load static %}
     <script src="{% static 'generator/seed.js' %}"></script>
     <link rel="stylesheet" href="{% static 'generator/background_select.css' %}">
+    {% if is_permalink %}
+        <meta property="og:title" content="Seed {{ share_id }} - Chrono Trigger: Jets of Time Randomizer" />
+        <meta property="og:description" content="{{ share_info }}" />
+        <meta property="og:type" content="website" />
+        <meta property="og:url" content="https://beta.ctjot.com{% url 'generator:share' share_id %}" />
+        <meta property="og:image" content="https://beta.ctjot.com{% url 'generator:seedimg' share_id %}" />
```

Comment:
I don't like the hardcoded beta.ctjot.com here, but we need absolute URLs. If you have a good suggestion for this I'm happy to hear it (maybe put it in a configuration file of some sort? unless Django gives us the domain name we're running on already, somewhere), otherwise we can change it when needed to leave beta, I guess.

Reply:
Well, at least og:url needs to be absolute. I think the image might actually work with just a domain-relative thing but it's better to have an absolute one, heh.